stacks-signer/src/v0/signer.rs

```diff
@@ -671,16 +666,30 @@ impl Signer {
     /// The actual `send_block_response` implementation. Declared so that we do
     /// not need to duplicate in testing.
     fn impl_send_block_response(&mut self, block_response: BlockResponse) {
-        let res = self
+        info!(
+            "{self}: Broadcasting a block response to stacks node: {block_response:?}";
+        );
+        let accepted = matches!(block_response, BlockResponse::Accepted(..));
+        let block_hash = block_response.get_signer_signature_hash();
+        match self
             .stackerdb
-            .send_message_with_retry::<SignerMessage>(block_response.clone().into());
-        match res {
-            Err(e) => warn!("{self}: Failed to send block rejection to stacker-db: {e:?}"),
-            Ok(ack) if !ack.accepted => warn!(
-                "{self}: Block rejection not accepted by stacker-db: {:?}",
-                ack.reason
-            ),
-            Ok(_) => debug!("{self}: Block rejection accepted by stacker-db"),
+            .send_message_with_retry::<SignerMessage>(block_response.into())
+        {
+            Ok(ack) => {
+                if !ack.accepted {
+                    warn!(
+                        "{self}: Block response not accepted by stacker-db: {:?}",
+                        ack.reason
+                    );
+                }
+                crate::monitoring::actions::increment_block_responses_sent(accepted);
+                if let Ok(Some(block_info)) = self.signer_db.block_lookup(&block_hash) {
```

Comment:
It looks like in all of the callers of this function, we already have the `block_info`, so maybe we could pass it in and avoid an extra DB lookup here?